ui-guidelines/css/guidelines.css

```diff
@@ -165,8 +159,58 @@ pre {
 	font-size: 1em;
 	font-weight: bold;
 }
-/*==== HELPERS ====*/
-.top-half-extra {margin-top: 2em;}
-.top-extra {margin-top: 4em;}
-.top-low {margin-top: 1.5em;}
-.bottom-half-extra {margin-bottom: 1.5em;}
\ No newline at end of file
+.btn {
+	display: none;
```

Comment:
I'm not sure, what's the point in having styles for elements like buttons with `display: none`? What is the use case?